pkg/storage/truncate.go

```diff
@@ -38,7 +40,7 @@ func (s *Store) TruncateAllTablesDANGEROUS(logger *zap.Logger) error {
 	trb_request
 	`
 
-	_, err := s.db.Exec("TRUNCATE " + tables)
+	_, err := s.db.Exec(fmt.Sprintf("TRUNCATE %s CASCADE", tables))
```

Comment:
Instead of cascade, could you add the table name to the table list? It's more explicit which tables we are dropping this way, instead of being inferred ( like you updated the dev script)

Reply:
Oh wow 🤦 thanks! I thought I added it here too and then ran into the postgres issue - I will add the table and make sure all is ok. Thanks!